src/without-bang-important.js

```diff
@@ -0,0 +1,23 @@
+// Module with the same interface as the core aphrodite module,
+// except that styles injected do not automatically have !important
+// appended to them.
+//
+import {injectAndGetClassName} from './inject';
+
+import {
+    StyleSheet,
+    StyleSheetServer,
+    StyleSheetTestUtils,
+} from './index.js'
+
+const css = (...styleDefinitions) => {
+    // Do not append !important to style definitions
+    return injectAndGetClassName(false, styleDefinitions);
```

Comment:
I just prefer to avoid comments where I can because they can go out of date. Also, unless you know the function signature, you don't know that this comment is referring to the `false` argument... It almost looks like a `todo` comment that's missing the `TODO` pragma.